application/modules/guest/views/invoices_index.php

```diff
@@ -21,76 +21,12 @@ class="btn  <?php echo $status == 'paid' ? 'btn-primary' : 'btn-default' ?>">
-                    <th><?php _trans('amount'); ?></th>
-                    <th><?php _trans('balance'); ?></th>
-                    <th><?php _trans('options'); ?></th>
-                </tr>
-                </thead>
-
-                <tbody>
-                <?php foreach ($invoices as $invoice) { ?>
-                    <tr>
-                        <td>
-                            <a href="<?php echo site_url('guest/invoices/view/' . $invoice->invoice_id); ?>">
-                                <?php echo $invoice->invoice_number; ?>
-                            </a>
-                        </td>
-                        <td>
-                            <?php echo date_from_mysql($invoice->invoice_date_created); ?>
-                        </td>
-                        <td>
-                            <?php echo date_from_mysql($invoice->invoice_date_due); ?>
-                        </td>
-                        <td>
-                            <?php _htmlsc(format_client($invoice)); ?>
-                        </td>
-                        <td>
-                            <?php echo format_currency($invoice->invoice_total); ?>
-                        </td>
-                        <td>
-                            <?php echo format_currency($invoice->invoice_balance); ?>
-                        </td>
-                        <td>
-                            <div class="options btn-group btn-group-sm">
-                                <?php if ($invoice->invoice_status_id != 4 && get_setting('enable_online_payments')) : ?>
-                                    <a href="<?php echo site_url('guest/payment_handler/make_payment/' . $invoice->invoice_url_key); ?>"
-                                       class="btn btn-primary">
-                                        <i class="fa fa-credit-card"></i>
-                                        <?php _trans('pay_now'); ?>
-                                    </a>
-                                <?php endif; ?>
-                                <a href="<?php echo site_url('guest/invoices/view/' . $invoice->invoice_id); ?>"
-                                   class="btn btn-default">
-                                    <i class="fa fa-eye"></i>
-                                    <?php _trans('view'); ?>
-                                </a>
-                                <a href="<?php echo site_url('guest/invoices/generate_pdf/' . $invoice->invoice_id); ?>"
-                                   class="btn btn-default">
-                                    <i class="fa fa-print"></i>
-                                    <?php _trans('pdf'); ?>
-                                </a>
-                            </div>
-                        </td>
-                    </tr>
-                <?php } ?>
-                </tbody>
-
-            </table>
-        </div>
+        <?php $this->layout->load_view('guest/invoices_partial_table', array('invoices' => $invoices)); ?>
```

Comment:
Please use the short array syntax using `[` and `]`.

Reply:
I do not understand

Reply:
Instead of `array('invoices' => $invoices)` use the short array syntax `['invoices' => $invoices]` as defined in the [contribution guide](https://github.com/InvoicePlane/InvoicePlane/blob/1.5/CONTRIBUTING.md#php).

Reply:
ready